docs/source/changelog.rst

```diff
@@ -26,6 +28,10 @@ Changelog
   no longer used to determine the time column. Now, both instance id columns and time columns in a cutoff time
   dataframe can be in any order as long as they are named properly.
 
+* The type_string attributes of all entity classes are now a snake case conversion of their class names. This
```

Comment:
* variable classes, not entity classes
* let's use code formatting (\`\`text\`\`) around type_string and the formal class names (Unknown, IPAddress, etc)
* Instead of talking about schema versions, in the last sentence let's say that old saved entitysets that used these variable may load incorrectly